code/game/objects/items/devices/scanners.dm

```diff
@@ -160,6 +160,24 @@ MASS SPECTROMETER
 	else if (M.getBrainLoss() >= 10)
 		user.show_message("<span class='warning'>Significant brain damage detected. Subject may have had a concussion.</span>", 1)
 
+	// Blood Level
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.vessel)
+			if(H.blood_max)
+				user.show_message("<span class='danger'>Subject is bleeding!</span>")
+			var/blood_volume = round(H:vessel.get_reagent_amount("blood"))

```

Comment:
For the record I don't blame you for the comma, it's baycode after all.

And yes, a period will work here, since H has already been initiated as a human.